pkg/lib/serverTime.js

```diff
@@ -258,42 +266,33 @@ export function ServerTime() {
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_timesync_backend() {
```

Comment:
In case cockpit is packaged by some distro which is not shipped in the list, this feature will be completely blocked. Is there a way to make this test dynamic, by testing the existance of some file on the filesystem?

Reply:
I couldn't think of a good one: https://github.com/cockpit-project/cockpit/pull/17992#issuecomment-1350611746

Reply:
On Arch Linux this is now disabled, I suppose we should default to `systemd-timesyncd` there. Maybe we should ask the SUSE folks what time server they use, but that's also fine for a follow up.

Reply:
We need to know which servuice is going to be activated while NTP is still off.

But we can improve the logic to say "Chronyd is installed and because this is Fedora, we expect chronyd to take precedence over timesyncd." (because we know how the actual chronyd RPM behaves.)  The admin can still change that, and then we show the "unexpected config" error.

Thus, we would still have a per-OS config, but it is smarter than just "always chronyd".

What about that?

Reply:
Right, so all OS would have the same configuration "chronyd if installed, otherwise timesyncd if installed, otherwise not supported", and we don't need any way to tweak that in the manifests. Nice.

Reply:
Thanks, this is a lot simpler and robust now!